src/EditorFeatures/Core/FindUsages/IFindUsagesContext.cs

```diff
@@ -23,6 +23,7 @@ internal interface IFindUsagesContext
 
         Task OnDefinitionFoundAsync(DefinitionItem definition);
         Task OnReferenceFoundAsync(SourceReferenceItem reference);
+        Task OnExternalReferenceFoundAsync(ExternalReferenceItem reference);
```

Comment:
Just to confirm: do we have any IVT implementations of this?

Reply:
we really really really really really shouldn't.  the idea of the context is that *we* create this and pass this out for *others* to populate.  the *impl* of this is literally the 'roslyn find refs window'.  If there was another impl, i have no idea how htat would work.

Reply:
Actually, i have no idea how vs4mac works.  @KirillOsenkov do you guys have an impl of IFindUsagesContext in your code?

Reply:
I presume they would, although they'd just be updating their implementation when they consume a new build from us.

Reply:
too bad i don't have DIMs.  This would be a perfect case to use them.  @KirillOsenkov for visibility.